src/main/java/iitema/gypsypokemon/elements/blocks/Abyss.java

```diff
@@ -26,41 +26,7 @@ protected boolean solid(Direction dir) {
      */
     @Override
     public boolean stepOn(Direction dir, PlayerInterface player) {
-        player.move(this);
         player.kill();
```

Comment:
If we kill the player the game will end before we move the player there... :( We first need to move the player above the abyss, then kill him